frontend/src/apiEndpoints.js

```diff
@@ -32,6 +32,13 @@ const apiEndpoints = {
     stream:{
         getAll: 'streams/all',
         getInfo: 'streams/info'
+    },
+
+    login:{
+        signUp: 'login/sign-up',
```

Comment:
login/sign-up sounds weird, doesn't it?

we should rename that controller to `user` probably or smth similar to that